### PR TITLE
feat: meetings

### LINKED ISSUE #125: [feat] space: meetings integration
this is the functionality implementation of meeting page, just like notes, the most recent meeting is opened always and in stack : meeting of the entire week.<br> user can choose from the stack to take a note.

meta data to show on nav:

- [ ] meeting title as note title
- [ ] url to join
- [ ] meeting color
- [ ] time & duration

data to show on stack view:

- [ ]  title, duration, time, url ( todo if any)

### PR DESCRIPTION
**Pull request by deewakar-k**
- [x] get meetings for current week on stack
- [x] get most recent meeting on space
- [x] meeting title as note title
- [x] url to join
- [x] select meetings from stack or toggle close.
- [x] write and save notes
- [x] change title

image

### CODE REVIEW

**Comment by immadsahin** [comment]
show the meeting url of opened one -just beside the duration/time— otherwise it looks empty

**Comment by deewakar-k** [comment]
> show the meeting url of opened one -just beside the duration/time— otherwise it looks empty

done!

File: apps/frontend/src/utils/meet.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      start: weekStart,
+      end: weekEnd,
+    })
+  )
+}
+
+export function formatMeetDate(date: string): string {
+  return format(new Date(date), "EEE, MMM dd")
+}
+
```
Comment (madhav-relish) [inline apps/frontend/src/utils/meet.ts:20]: Can be generic functions which can be used for other things too not just for meets like
formatDate, formatTimeRange etc

**Review by madhav-relish — DISMISSED** [review]

**Comment by sajdakabir** [comment]
@deewakar-k , Please update to match the main branch and fix the build error

**Comment by immadsahin** [comment]
@deewakar-k 
https://github.com/user-attachments/assets/b1fcad80-8a7a-4dea-8978-983836efba81

it scrolls the whole page, only list of meetings should scroll.

- [ ]on 13 inch mac:  the font of meeting titles looks bigger.
- need more gap between meetings card
- need border on hover.
- make google meet url dull, glow on hover ( in meeting list stack)
- 
Screenshot 2024-10-11 at 1 36 40 PM
- text area overlaping with title
Screenshot 2024-10-11 at 1 38 10 PM
-it still shows google meet url, it should show the proper url

**Pull request merged**